src/main/java/carriermanagementsystem/CarrierManagementSystemClient.java

```diff
@@ -68,27 +68,38 @@ public static void main(String[] args) {
 //         * case #2: pass null for the RfPorts 
 //         */
 //        
-//        FrequencyBand WcdmaBand_6 = FrequencyBand.WCDMA_BAND_6;
+//        FrequencyBand WcdmaBand_5 = FrequencyBand.WCDMA_BAND_5;
```

Comment:
Pardon, to quickly clarify, was there a reason for changing these bands in response to a point of feedback Ista had gave? 

Or was this for testing?

Reply:
I realized the variable names just didn't match up with the examples so I changed them back